src/datajudge/constraints/base.py

```diff
@@ -113,7 +113,16 @@ class Constraint(abc.ABC):
     """
 
     def __init__(
-        self, ref: DataReference, *, ref2=None, ref_value: Any = None, name: str = None
+        self,
+        ref: DataReference,
+        *,
+        ref2=None,
+        ref_value: Any = None,
+        name: str = None,
+        output_postprocessing_sorter: Callable[
```

Comment:
Is this only capable of doing sorting? Probably you could do any kind of postprocessing of the output. As such, I'd suggest changing the name of the argument to `output_processor`or if you want to be more precise `output_postprocessor`.

Reply:
Done